### PR TITLE
{bio}[foss/2016a,intel/2016a] SNPhylo v20140701

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

requires ~~https://github.com/hpcugent/easybuild-easyblocks/pull/865~~

depends on ~~#2693~~, ~~#2694~~, ~~#2695~~, ~~#2697~~, ~~#2698~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8f38a8aea72323080ff1 for a full test report.

File: easybuild/easyconfigs/s/SNPhylo/SNPhylo-20140701-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+version = '20140701'
+
+homepage = 'http://chibba.pgml.uga.edu/snphylo/'
+description = """SNPhylo: a pipeline to generate a phylogenetic tree from huge SNP data"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+# download from http://chibba.pgml.uga.edu/snphylo/snphylo.tar.gz and rename
+# see http://chibba.pgml.uga.edu/snphylo/ for version
+sources = ['%(namelower)s-%(version)s.tar.gz']
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SNPhylo/SNPhylo-20140701-foss-2016a.eb]: `sourcelower_tar_gz`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2715.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/004c56eac87d47323cd2 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
going in, thanks again @wpoely86!

**Pull request merged**